### PR TITLE
Update reference capture regex && Add Jest unit test

### PR DESCRIPTION
**Pull request by zhx828**
This fixes https://github.com/oncokb/oncokb/issues/3599

### CODE REVIEW

File: jest.conf.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const tsconfig = require('./tsconfig.json');
```
Comment (zhx828) [inline jest.conf.js:1]: I copied the file from oncokb-transcript.

File: src/main/webapp/app/config/constants/regex.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// example reference groups to capture:
+//     (PMID: 11900253)
+//     (PMID: 11753428, 16007150, 21467160)
+//     (NCT1234567)
+export const REF_CAPTURE = /(\(\s*(?:PMID|NCI|Abstract):.*?\))/i;
```
Comment (calvinlu3) [inline src/main/webapp/app/config/constants/regex.ts:5]:
```suggestion
export const REF_CAPTURE = /(\(\s*(?:PMID|NCT|Abstract):.*?\))/i;
```

File: src/main/webapp/app/config/constants/regex.spec.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { REF_CAPTURE } from 'app/config/constants/regex';
+
+describe('Regex constants test', () => {
```
Comment (calvinlu3) [inline src/main/webapp/app/config/constants/regex.spec.ts:3]: Add another test for NCT?

**Review by calvinlu3 — APPROVED** [review]
LGTM

**Pull request merged**